packages/world/ts/getWorldAbi.ts

```diff
@@ -0,0 +1,27 @@
+import { Client, Abi, Address, getAddress } from "viem";
+import IBaseWorldAbi from "../out/IBaseWorld.sol/IBaseWorld.abi.json";
+import { functionSignatureToAbiItem } from "./functionSignatureToAbiItem";
+import { getFunctions } from "./getFunctions";
+
+export async function getWorldAbi({
+  client,
+  worldAddress,
+  fromBlock,
+  toBlock,
```

Comment:
we could probably default this
```suggestion
  toBlock = 'latest',
```

Reply:
Agreed, can change to it after trying it out with world explorer.